test/integration/execute-query-multilingual-test.js

```diff
@@ -0,0 +1,118 @@
+import PathProxy from '../../src/PathProxy';
+import JSONLDResolver from '../../src/JSONLDResolver';
+import LanguageResolver from '../../src/LanguageResolver';
+import { createQueryEngine } from '../util';
+import { namedNode, literal } from '@rdfjs/data-model';
+import defaultHandlers from '../../src/defaultHandlers';
+import LanguageHandler from '../../src/LanguageHandler';
+
+import context from '../context';
+
+const subject = namedNode('https://example.org/#me');
+const queryEngine = createQueryEngine([
+  literal('Tomato', 'en'),
+  literal('Second Tomato', 'en'),
+  literal('Tomaat', 'nl'),
+  literal('Tomate', 'de'),
+  literal('zweite Tomate', 'de'),
+]);
+
+describe('querying a language with @langcode', () => {
+  let tomato;
+  beforeAll(() => {
+    const resolvers = [
+      new LanguageResolver(context),
+      new JSONLDResolver(context),
+    ];
+
+    const pathProxy = new PathProxy({ handlers: defaultHandlers, resolvers });
+    tomato = pathProxy.createPath({ queryEngine }, { subject });
+  });
+
+  it('returns results in the selected language', async () => {
+    const dutchLabel = await tomato.label['@nl'];
+    expect(`${dutchLabel}`).toEqual('Tomaat');
+
+    const englishLabel = await tomato.label['@en'];
+    expect(`${englishLabel}`).toEqual('Tomato');
+
+    const germanLabel = await tomato.label['@de'];
+    expect(`${germanLabel}`).toEqual('Tomate');
+  });
+});
+
+describe('querying a language with the language handler', () => {
+  let tomato;
+  beforeAll(() => {
+    const resolvers = [
+      new LanguageResolver(context),
+      new JSONLDResolver(context),
+    ];
+
+    const pathProxy = new PathProxy({ handlers: { ...defaultHandlers,
+      en: new LanguageHandler('en'),
+      de: new LanguageHandler('de'),
+      nl: new LanguageHandler('nl'),
+    }, resolvers });
+    tomato = pathProxy.createPath({ queryEngine }, { subject });
+  });
+
+  it('returns results in the selected language via the language handler', async () => {
+    const dutchLabel = await tomato.label.nl;
+    expect(`${dutchLabel}`).toEqual('Tomaat');
+
+    const englishLabel = await tomato.label.en;
+    expect(`${englishLabel}`).toEqual('Tomato');
+
+    const germanLabel = await tomato.label.de;
+    expect(`${germanLabel}`).toEqual('Tomate');
+  });
+});
+
+describe('language defaults', () => {
+  let tomato;
+  beforeAll(() => {
+    context['@context']['@language'] = 'de';
+
+    const resolvers = [
+      new LanguageResolver(context),
+      new JSONLDResolver(context),
+    ];
+
+    const pathProxy = new PathProxy({ handlers: { ...defaultHandlers,
+      en: new LanguageHandler('en'),
```

Comment:
I think it would be useful to have the following tests work

```ts
  it('returns results in the selected language via the language handler', async () => {
    const dutchLabel = await tomato.label.nl;
    expect(`${dutchLabel}`).toEqual('Tomaat');

    const englishLabel = await tomato.label.en;
    expect(`${englishLabel}`).toEqual('Tomato');

    const germanLabel = await tomato.label.de;
    expect(`${germanLabel}`).toEqual('Tomate');
  });
```

Even though the `de` default is given in the context.

Reply:
Do these not work on your side?
For me they work.

Reply:
Very good point. It failed there. I added more test and fixed the problem.